src/components/sellerPages/inventory/current/index.js

```diff
@@ -17,18 +17,15 @@ function CurrentInventory({ inventory, fetchProducts, getProductsStatus }) {
     fetchProducts(authState);
   }, []);
 
-  const displayedData = useSearch(inventory, 'name', searchData);
-  console.log(inventory);
   return (
     <>
       <NavBar searchVisible={false} setData={setSearchData} />
       <div className="outerContainer">
         <div className="contents">
-          <SearchResults data={displayedData} filter={searchData} />
+          <SearchResults data={inventory} filter={searchData} />
```

Comment:
really a headache to figure out what was making the 2nd call here but we got it, nicely done